utils/dc-chain/patches/newlib-4.3.0.20230120-kos.diff

```diff
@@ -243,374 +243,11 @@ diff --color -ruN newlib-4.3.0.20230120/newlib/libc/stdlib/assert.c newlib-4.3.0
  #ifndef HAVE_ASSERT_FUNC
  /* func can be NULL, in which case no function information is given.  */
  void
-@@ -72,3 +74,5 @@
+@@ -72,3 +74,7 @@
     __assert_func (file, line, NULL, failedexpr);
    /* NOTREACHED */
  }
 +#endif
-+
-diff --color -ruN newlib-4.3.0.20230120/newlib/libc/sys/sh/ftruncate.c newlib-4.3.0.20230120-kos/newlib/libc/sys/sh/ftruncate.c
---- newlib-4.3.0.20230120/newlib/libc/sys/sh/ftruncate.c	2023-02-02 15:26:13.675093134 -0600
-+++ newlib-4.3.0.20230120-kos/newlib/libc/sys/sh/ftruncate.c	2023-02-02 15:26:28.806127897 -0600
-@@ -1,9 +1,2 @@
--#include <_ansi.h>
--#include <sys/types.h>
--#include "sys/syscall.h"
-+/* Nothing here */
- 
--int
--ftruncate (int file, off_t length)
--{
--  return __trap34 (SYS_ftruncate, file, length, 0);
--}
-diff --color -ruN newlib-4.3.0.20230120/newlib/libc/sys/sh/sys/lock.h newlib-4.3.0.20230120-kos/newlib/libc/sys/sh/sys/lock.h
---- newlib-4.3.0.20230120/newlib/libc/sys/sh/sys/lock.h	1969-12-31 18:00:00.000000000 -0600
```

Comment:
This has been placed into sys/lock.h inside of KOS. The patch.mk file now copies it to this location, as it was already doing for other files.